src/api/Controllers/CodeListController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using BDMS.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Reflection;
+
+namespace BDMS.Controllers;
+
+[ApiController]
+[Route("api/v{version:apiVersion}/[controller]")]
+public class CodeListController : ControllerBase
+{
+    public BdmsContext Context { get; }
+
+    public CodeListController(BdmsContext context)
+    {
+        Context = context;
+    }
+
+    /// <summary>
+    /// Asynchronously gets the <see cref="Codelist"/>s, optionally filtered by <paramref name= "schema"/>.
```

Comment:
PP: Unnötige Whitespaces in XML-Dokumentation entfernen
```suggestion
    /// Asynchronously gets the <see cref="Codelist"/>s, optionally filtered by <paramref name="schema"/>.
```